prover/circuit_synthesizer/src/metrics.rs

```diff
@@ -0,0 +1,15 @@
+use std::time::Duration;
+use vise::{Buckets, Histogram, LabeledFamily, Metrics};
+
+#[derive(Debug, Metrics)]
+#[metrics(prefix = "prover_circuit_synthesizer")]
+pub(crate) struct CircuitSynthesizerMetrics {
+    #[metrics(buckets = Buckets::LATENCIES, labels = ["blob_size_in_gb"])]
+    pub blob_sending_time: LabeledFamily<u64, Histogram<Duration>>,
+    #[metrics(buckets = Buckets::LATENCIES, labels = ["circuit_type"])]
+    pub synthesize: LabeledFamily<&'static str, Histogram<Duration>>,
+}
+
+#[vise::register]
+pub(crate) static CIRCUIT_SYNTHESIZER_METRICS: vise::Global<CircuitSynthesizerMetrics> =
```

Comment:
Bikeshedding: If a crate uses a single metrics type, IMO, the corresponding static could be safely called just `METRICS` – there are no other metrics to confuse them with.